package.json

```diff
@@ -18,7 +18,8 @@
     "node": ">=0.10.0"
   },
   "scripts": {
-    "test": "xo && node test.js"
+    "test": "xo && nyc node test.js",
+    "coverage": "nyc --reporter=text-lcov node test.js | coveralls"
```

Comment:
Why run the test again here? You already have the report from `test`.

According to the `nyc` docs it should be: `"coverage": "nyc report --reporter=text-lcov | coveralls",`?